Updating SWRL.md

Vardhini999 opened the pull request.

Vardhini999 committed:
Updated SWRL.md

Vardhini999 committed:
Updated SWRL.md

Vardhini999 committed:
Updated SWRL.md

Vardhini999 committed:
Updated SWRL.md

cogan-shimizu commented:
* What do the "|" symbols mean in your syntax example? Explain in the same paragraph where you define the symbols.
* Unify your use of the -> vs unicode symbol arrow.

Vardhini999 committed:
Modified construct of atoms and used unified Unicode arrow

Vardhini999 commented:
> * What do the "|" symbols mean in your syntax example? Explain in the same paragraph where you define the symbols.
> * Unify your use of the -> vs unicode symbol arrow.

* I just meant to put them together as a table, but it turned out to be different from expected. Removed them now.
* It was a typo, updated the unicode arrow → everywhere now.
